src/TitleDetails/BoundItemsList/hooks/useBoundItems/useBoundItems.js

```diff
@@ -20,15 +20,27 @@ export const useBoundItems = ({ titleId, poLineId, options = {} }) => {
   const { enabled = true, ...otherOptions } = options;
 
   const stripes = useStripes();
-  const ky = useOkapiKy();
   const [namespace] = useNamespace({ key: 'bound-items-list' });
 
   const {
     activeTenantId,
     isCentralOrderingEnabled,
+    targetTenantId,
+    isTargetTenantCentral,
+    centralTenantId,
   } = useReceivingSearchContext();
+  const ky = useOkapiKy({ tenant: targetTenantId });
 
-  const boundItemsQuery = `titleId==${titleId} and poLineId==${poLineId} and isBound==true`;
+  let boundItemsQuery = `titleId==${titleId} and poLineId==${poLineId} and isBound==true`;
+  const showActiveTenantItems = (
+    isCentralOrderingEnabled
```

Comment:
we should show bind items even without centralOrderingEnabled.

Joe Reimers
  Yesterday at 7:40 PM
Eventually it shouldn't matter, but we need to make sure that when central ordering is disabled, a member tenant can order and bind (this is what MOBIUS and Galileo will do)